Content.Server/AlertLevel/AlertLevelSystem.cs

```diff
@@ -61,39 +61,41 @@ private void OnStationInitialize(StationInitializedEvent args)
         var defaultLevel = alertLevelComponent.AlertLevels.DefaultLevel;
         if (string.IsNullOrEmpty(defaultLevel))
         {
-            defaultLevel = alertLevelComponent.AlertLevels.Levels.Keys.First();
+            defaultLevel = alertLevelComponent.AlertLevels.Levels.First(); // WL-Changes: Alert Level Rework
         }
 
         SetLevel(args.Station, defaultLevel, false, false, true);
     }
 
     private void OnPrototypeReload(PrototypesReloadedEventArgs args)
     {
-        if (!args.ByType.TryGetValue(typeof(AlertLevelPrototype), out var alertPrototypes)
-            || !alertPrototypes.Modified.TryGetValue(DefaultAlertLevelSet, out var alertObject)
-            || alertObject is not AlertLevelPrototype alerts)
-        {
+        // WL-Changes-start: Alert Level Rework
+        if (!args.ByType.TryGetValue(typeof(AlertLevelsListPrototype), out var alertListPrototypes)
+            || !alertListPrototypes.Modified.TryGetValue(DefaultAlertLevelSet, out var alertObject)
+            || alertObject is not AlertLevelsListPrototype alerts)
             return;
-        }
+        // WL-Changes-end
 
         var query = EntityQueryEnumerator<AlertLevelComponent>();
         while (query.MoveNext(out var uid, out var comp))
         {
             comp.AlertLevels = alerts;
 
-            if (!comp.AlertLevels.Levels.ContainsKey(comp.CurrentLevel))
+            if (!comp.AlertLevels.Levels.Contains(comp.CurrentLevel)) // WL-Changes: Alert Level Rework
             {
                 var defaultLevel = comp.AlertLevels.DefaultLevel;
-                if (string.IsNullOrEmpty(defaultLevel))
+                if (string.IsNullOrEmpty(defaultLevel) && comp.AlertLevels.Levels.Count > 0)
```

Comment:
```suggestion
                if (string.IsNullOrEmpty(defaultLevel) /*WL-Changes-start*/&& comp.AlertLevels.Levels.Count > 0/*WL-Changes-end*/)
```

забыта обертка изменений в wl-changes теги